helm_deploy/hmpps-prisoner-to-nomis-update/templates/report-appointments-cronjob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-report-appointments
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "5 0 * * THU"
```

Comment:
any reason not to be daily?

Reply:
Just being cautious as it is big and slow atm, though with Mikes comments about just doing a few days worth that would change